components/file_preview_modal/image_preview.tsx

```diff
@@ -1,14 +1,20 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 
+import {FileInfo} from '@mattermost/types/files';
+
 import {getFilePreviewUrl, getFileDownloadUrl} from 'mattermost-redux/utils/file_utils';
 
 import './image_preview.scss';
 
-export default function ImagePreview({fileInfo, canDownloadFiles}) {
+export interface Props {
```

Comment:
if we are not going to import this prop, we might as well not export it

Reply:
oh, my bad. I've fixed

P.S. Happy new year :)

Reply:
Happy new year 🥳